index.rst

```diff
@@ -492,14 +492,21 @@ What's new in PyTorch tutorials?
    :image: _static/img/thumbnails/cropped/60-min-blitz.png
    :link: advanced/static_quantization_tutorial.html
    :tags: Quantization
-   
+
 .. customcarditem::
    :header: Grokking PyTorch Intel CPU Performance from First Principles
    :card_description: A case study on the TorchServe inference framework optimized with Intel® Extension for PyTorch.
    :image: _static/img/thumbnails/cropped/generic-pytorch-logo.png
    :link: intermediate/torchserve_with_ipex
    :tags: Model-Optimization,Production
 
+.. customcarditem::
+   :header: Multi-Objective Neural Architechture Search with Ax
+   :card_description: Learn how to use Ax to search over architectures find optimal tradeoffs between accuracy and latency.
+   :image: _static/img/ray-tune.png
+   :link: beginner/ax_multiobjective_nas_tutorial.html
+   :tags: Model-Optimization,Best-Practice
```

Comment:
might be nice to add an Ax and TorchX tag here, seems like that's what TorchRec does

Reply:
```suggestion
   :tags: Model-Optimization,Best-Practice,Ax,TorchX
```